geo.txt

```diff
@@ -56,3 +56,1703 @@ Phys 91(11):957-962
+
+B H   1.197(2)
+H B H 129.6(2)
+
+NAME = BH6N
+# Point Group: C3v
+# Shape: Trigonal pyramidal --- Trigonal pyramidal
+# Bond lengths: rm^(1)     (angstroms)
+# Bond angles:  thetam^(1) (degrees)
+# Source Paper: Demaison J, Lievin J, Császár AG, Gutle C (2008) Equilibrium structure and torsional barrier of BH3NH3. J
+Phys Chem A 112(19):4477-4482
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+B...N   1.6576(16)
+B   H   1.2098(2)
+N   H   1.0083(3)
+N...B H 104.73(1)
+B...N H 110.29(2)
+
+# Bond lengths: re^se     (angstroms)
+# Bond angles:  thetae^se (degrees)
+
+B...N   1.6453(1)
+B   H   1.2058(3)
+N   H   1.0101(3)
+N...B H 105.00(2)
+B...N H 110.97(2)
+
+NAME = BeI2
+# Point Group: Dooh
+# Shape: Bent
+# Bond lengths: rg     (angstroms)
+# Bond angles:  thetag (degrees)
+# Source Paper: Shlykov SA, Zhabanov YA, Giricheva NI, Girichev AG, Girichev GV (2015) Combined gas electron
+diffraction/mass spectrometric study of beryllium diiodide assisted by quantum chemical calculations: structure
+and thermodynamics of beryllium dihalides. Struct Chem 26 (5-6):1451-1458
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+# Temperature = 501(5) Kelvin
+
+Be I   2.163(6)
+I  Be I 165(3)
+
+# Temperature = 722(10) Kelvin
+
+Be I   2.172(7)
+I  Be I 162(4)
+
+NAME = BrGeH
+# Point Group: Cs
+# Shape: Bent
+# Bond lengths: r0     (angstroms)
+# Bond angles:  theta0 (degrees)
+# Source Paper: Kang L, Sunahori F, Minei AJ, Clouthier DJ, Novick SE (2009) Fourier transform microwave spectroscopy of
+monobromogermylene (HGeBr and DGeBr), a heavy atom carbene analog. J Chem Phys 130(12):124317/1-
+124317/11
+doi: 10.1063/1.3080161
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Ge Br   2.3276(9)
+Ge H    1.6093(7)
+H Ge Br 92.44(4)
+
+# Bond lengths: rm(1)     (angstroms)
+# Bond angles:  thetam(1) (degrees)
+
+Ge Br   2.3243(2)
+Ge H    1.563(9)
+H Ge Br 93.55(24)
+
+NAME = BrHO 2010
+# Point Group: Cs
+# Shape: Bent
+# Bond lengths: re     (angstroms)
+# Bond angles:  thetae (degrees)
+# Source Paper: Cohen EA, Müller HSP, Tan TL, McRae GA (2010) High resolution spectroscopy of DOBr and molecular
+properties of hypobromous acid. J Mol Spectrosc 262(1):30-36
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+H O    0.9640(3)
+O Br   1.82800(1)
+H O Br 102.99(1)
+
+NAME = Br2Fe
+# Point Group: Dooh
+# Shape: Bent
+# Bond lengths: rg     (angstroms)
+# Bond angles:  thetaa (degrees)
+# Source Paper: a. Varga Z, Kolonits M, Hargittai M (2011) Iron dihalides: Structures and thermodynamic properties from
+computation and an electron diffraction study of iron diiodide. Struct Chem 22 (2):327-336 b. Hargittai M, Subbotina NY, Kolonits M, Gershikov AG (1991) Molecular structure of first-row transition
+metal dihalides from combined electron diffraction and vibrational spectroscopic analysis. J Chem Phys 94:
+7278-7286.
+# Date first added: 2023-07-05 (by Sichao Xiong)
+
+Fe   Br    2.289(8)
+Br...Br    4.476(12)
```

Comment:
I see that Vogt's book says Br...Br, but we have been using that notation for Van der Waals interactions. 

<s>I'm also confused because the Br...Br length is not double the FeBr lenght. It's as if they have two different geometries here: one based on Fe-Br vibrations (in which case we can infer the Br-Br distance by multiplying by 2), and one based on Br-Br vibrations (in which we can infer the Fe-Br distance by dividing by 2). </s>

I see that the book also says that there's a comment to see, next to where it says the point group is Dooh, and that comment is here:

![image](https://github.com/HPQC-LABS/goDatabase/assets/1163013/d4e6364e-0714-43a1-9021-5251d56ddede)

At the beginning of the book, r_g is defined as "Thermal average value of internuclear distance", so the rg values along with the theta_a value would go under C2v, and the r_e value would instead go under Dooh. This would explain the lack of symmetry in what I described above and <s>struck out</s>.

Reply:
Yes, that would explain it because using the cosine rule on the angle Br–Fe–Br 155.9(29) and side 2.289(8) gives us approximately 4.476(12). So how should we deal with the molecules that have (see comment) next to the geometry?